core/src/test/java/org/elasticsearch/script/ScriptTests.java

```diff
@@ -52,6 +54,21 @@ public void testScriptParsing() throws IOException {
         }
     }
 
+    public void testScriptSimpleParsing() throws IOException {
+        String simpleScript = "{\"script\":\"_score + doc['field']\"}";
```

Comment:
No, this shorthand version is still supported.  We'd have to remove it if https://github.com/elastic/elasticsearch/issues/20122 goes in